src/js_native_api_v8.h

```diff
@@ -39,6 +39,10 @@ struct napi_env__ {
   inline void Unref() { if ( --refs == 0) delete this; }
 
   virtual bool can_call_into_js() const { return true; }
+  virtual v8::Maybe<bool> mark_arraybuffer_as_untransferable(
+      v8::Local<v8::ArrayBuffer> ab) const {
```

Comment:
nit: unused ?

Reply:
@devnexen This method is called in `js_native_api_v8.cc` (right above this file in the github diff)

Reply:
In fact I mentioned the argument :-)

Reply:
Yeah, the argument is not used here, only in the subclass override